v2/protegrity-data-tokenization/src/main/java/com/google/cloud/teleport/v2/templates/ProtegrityDataTokenization.java

```diff
@@ -206,6 +208,11 @@ public static void main(String[] args) {
         .withValidation()
         .as(DataflowPipelineOptions.class);
```

Comment:
We need have workaround about `org.apache.beam.runners.dataflow.options.DataflowPipelineOptions#getProject` option, that required for Dataflow runner, but not have sense for DirectRunner.

Reply:
Thank you for notice. I'll fix it.

Reply:
@MikhailMedvedevAkvelon does `getProject` throw an error in case if the project isn't specified?

Reply:
Rewrited this piece of code. Can you see, please? 
If it's DataflowRunner, we use DataflowPipelineOptions, whereas we deal with DirectRunner we don't need it.